cookbooks/bcpc-hadoop/templates/default/hdp_mapred-site.xml.erb

```diff
@@ -11,13 +11,13 @@
 <configuration>
   <property>
     <name>mapreduce.admin.map.child.java.opts</name>
-    <value>-server -Djava.net.preferIPv4Stack=true -Dhdp.version=2.2.0.0-2041</value>
+    <value>-server -Djava.net.preferIPv4Stack=true -Dhdp.version=<%= node[:bcpc][:hadoop][:distribution][:release] %></value>
     <final>true</final>
   </property>
 
   <property>
     <name>mapreduce.admin.user.env</name> 
-    <value>LD_LIBRARY_PATH=/usr/hdp/current/hadoop-client/lib/native:/usr/hdp/current/hadoop-client/lib/native/Linux-amd64-64</value>
+    <value>LD_LIBRARY_PATH=/usr/hdp/<%= node[:bcpc][:hadoop][:distribution][:release] %>/hadoop-client/lib/native:/usr/hdp/<%= node[:bcpc][:hadoop][:distribution][:release] %>/hadoop-client/lib/native/Linux-amd64-64</value>
```

Comment:
Please update this line to include a [fix](https://bbgithub.dev.bloomberg.com/BACH-Clusters/DNJ2-GEN/pull/5).